pipeline/api/cloud.py

```diff
@@ -27,9 +27,16 @@ class PipelineCloud:
     token: Optional[str]
     url: Optional[str]
 
-    def __init__(self, url: str = None) -> None:
-        self.token = os.getenv("PIPELINE_API_TOKEN")
+    def __init__(self, url: str = None, token: str = None) -> None:
+        self.token = token or os.getenv("PIPELINE_API_TOKEN")
         self.url = url or os.getenv("PIPELINE_API_URL", "https://api.pipeline.ai")
+        if self.token is not None:
+            self.authenticate()
+        else:
+            print(
+                "No token set, please set one and invoke ",
+                "PipelineCloud.authenticate() method",
+            )
```

Comment:
This will result in a double space

```
>>> print("abc ", "123")
abc  123
>>> print("abc ", "123")
abc 123
```

either

```suggestion
            print(
                "No token set, please set one and invoke",
                "PipelineCloud.authenticate() method",
            )
```

or

```suggestion
            print(
                "No token set, please set one and invoke "
                "PipelineCloud.authenticate() method"
            )
```